django/db/backends/sqlite3/operations.py

```diff
@@ -21,6 +21,10 @@ class DatabaseOperations(BaseDatabaseOperations):
         'DateTimeField': 'TEXT',
     }
     explain_prefix = 'EXPLAIN QUERY PLAN'
+    # Datatypes that cannot be extracted properly, due to the way the
```

Comment:
"These are the types of data that ..." (PEP 8 wants comments to be complete sentences).